### ISSUE TITLE
Primary index is missing

### ISSUE THREAD

**Issue by ghost** [matt-cook/osis-bibles#5]
I'm a user of a Sailfish OS app called "Bible'me". Unfortunately it doesn't work anymore since you deleted the `/index.xml` ( #f3367ec1de976653cfc677904c3c2a851eeef2b2) Maybe this leads to an error with other bible apps as well, so I wanted to ask if you could bring this file back.

Please excuse me, if I don't know all conventions on reporting issues on Github and everything yet. Thanks for this awesome repo!

**Comment by sebma** [comment]
Yep, I lost somehow some of my data on my SailFish OS and now I cannot download your bibles from [Bible'me](http://gitlab.unique-conception.org/apps-4-sailfish/bible-me) either.

It's a shame because you are the only one I know that provide us with the "kjf" (King James Francaise) translation.

Can you please fix this ?

Jesus Christ bless you.

**Comment by sebma** [comment]
In the meantime, the `index.xml` can be downloaded from [there](https://gist.github.com/jasonmoofang/c34fe71ea1800714f6ae8d9510843e52#file-index-xml) and must be put in `~/.local/share/harbour-bibleme/harbour-bibleme/`

And then, restart `bibleme`

**Comment by Olf0** [comment]
Oh, I see `index.xml`was created with [this commit (2013-11-09)](https://github.com/matt-cook/bible/commit/faa0238344d6e7737dfacfdc44a8a4e01c8894c3#diff-7323624b83e1dea8ba3cdf1fdfeb8d5c) and existed unchanged until it was deleted with [that commit (2016-03-05)](https://github.com/matt-cook/bible/commit/f3367ec1de976653cfc677904c3c2a851eeef2b).  It is (thanks git) still available [in its original state](https://github.com/matt-cook/bible/blob/455e1913afae458445b49102d3e0375158dfba03/index.xml) (i.e. as between 2013-11-09 and 2016-03-05). 

@matt-cook, can you please recreate `index.xml` (or restore the old one, if it still corresponds to the available bibles) or share why it was removed, as its absence renders the software Bible'me (and all other software relying on the existence of `index.xml`) useless. 

P.S.: @sebma, thank you for providing an easy download to work around this issue. 
      Still manually downloading the ZIP file, extracting its contents and putting `index.xml` in the right place is tedious and establishes a significant technical hurdle for many potential bible readers.
